src/lockscreen.js

```diff
@@ -41,9 +41,8 @@ var Lockscreen = GObject.registerClass({
         this._activeSoundID = null;
         this._updateUI();
 
-        this._playbin.connect('realize', () => {
-            this._playActiveSound();
-        });
+        this._manager.connect_after('changed', this._playActiveSound.bind(this));
+        this._playbin.connect('realize', this._playActiveSound.bind(this));
```

Comment:
Why do we need to start the sound on both these places?

Also there seems to be a regression where the sound does not stop after the animation starts (I can reporduce it on the system app)

Reply:
> Why do we need to start the sound on both these places?

This is what Martin added. I am not totally sure TBH if `        this._manager.connect_after('changed', this._playActiveSound.bind(this));` should be considered.

> Also there seems to be a regression where the sound does not stop after the animation starts (I can reporduce it on the system app)

Are you testing it against https://github.com/endlessm/hack-sound-server/pull/55 ?

Reply:
@xjuan can you take another look with the #55 sound server?

Reply:
Yes I did,, see comment in phabricator